app/src/test/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivityLocalTest.kt

```diff
@@ -216,6 +263,27 @@ class QuestionPlayerActivityLocalTest {
     return scrollToHolder(StateViewHolderTypeMatcher(viewType))
   }
 
+  // Returns a matcher that matches for active confetti.
+  private fun hasActiveConfetti(): ViewAssertion {
+    return StateFragmentActiveConfettiAssertion()
+  }
+
+  // Custom class to check if a KonfettiView isActive().
+  private class StateFragmentActiveConfettiAssertion() : ViewAssertion {
+    override fun check(view: View, noViewFoundException: NoMatchingViewException?) {
+      if (noViewFoundException != null) {
+        throw noViewFoundException
+      }
+      check(view is KonfettiView) { "The asserted view is not KonfettiView" }
+
+      assertThat(
```

Comment:
Please actually call this on the object rather than directly importing it since it's very close to Truth (which we use everywhere else), and I don't think we want to encourage using it elsewhere.

That being said, this assertion class might better fit in a shared utility to avoid duplicating it between question & state players. I suggest moving it to a class in the testing module if possible. Ditto for the convenience method above, as well.

Reply:
Done, created a `KonfettiViewMatcher` class